File: LeetCode/1. 两数之和.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        nums_table = dict()
        for i in range(0, len(nums)):
            nums_table[nums[i]] = i
        for i in range(0, len(nums)):
            if ((target - nums[i]) in nums_table) and i != nums_table.get(target - nums[i]):
                return [i, nums_table.get(target - nums[i])]
        return []

File: LeetCode/1. 两数之和.cpp
#include <vector>
#include <algorithm>
using namespace std;

class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> ans;

        for(int i = 0; i < nums.size(); i++) 
            for(int j = i + 1; j < nums.size(); j++) {
            if(target - nums[i] == nums[j]){
                ans.emplace_back(i);
                ans.emplace_back(j);
                return ans;
            }
        }
        return ans;
    }
};